tools/viewbridge-spike/run.sh

```
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Orchestrate one VB spike run: install the launchd broker, launch the service
+# child, launch the host, stream ViewBridge os_log, then tear everything down.
+set -uo pipefail
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)